hive/hive-slow-reducer.md

```diff
@@ -35,13 +35,13 @@ Open [beeline](https://docs.microsoft.com/en-us/azure/hdinsight/hadoop/apache-ha
 
 The value of this variable is meant to be set to true/false based on the nature of the data.
 
-If the partitions in the input table is less(say less than 10), and so is the the number of output partitions, and the variable is set to ```true```, this causes data to be globally sorted and written using a single reducer. Even if the number of reducers available is larger, only a handful will be doing the real work and the max parallelism cannot be attained. When changed to ```false```, each reducer will write output into the output table and will be faster. In this process however many files will be written into the output table, which might affect the subsequest query performance. 
+If the partitions in the input table is less(say less than 10), and so is the the number of output partitions, and the variable is set to ```true```, this causes data to be globally sorted and written using a single reducer. Even if the number of reducers available is larger, a few reducers may be lagging behind due to data skew and the max parallelism cannot be attained. When changed to ```false```, more than one partitions may be handled by a single reducer and multiple smaller files will be written out, resulting in faster insert. This might affect further queries though because of the presence of smaller files. 

```

Comment:
written using a single reducer --> written using a single reducer per partition
more than one partitions may be handled by a single reducer --> more than one reducer may be handle a single partition